modules/cco-ccoctl-install-verifying.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
```

Comment:
Please consider moving this module to https://60222--docspreview.netlify.app/openshift-enterprise/latest/installing/validating-an-installation.html#validating-an-installation. @bscott-rh and I are trying to make a concerted effort to not repeat content over and over in installation assemblies. While in some cases it cannot be avoided, I think for this verification step, we can. You could always put a link to it under Next steps at the bottom of each assembly.